reports/aksversions/README.md

```diff
@@ -8,50 +8,62 @@ The scripts goal is to find every AKS cluster within a set of subscriptions, fin
 
 There are 2 scripts that make this report work:
```

Comment:
Want to include info about how set_env is being used / why it's needed too? May help in the future